modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/ProjectReadAccessType.java

```diff
@@ -0,0 +1,24 @@
+package com.enonic.xp.admin.impl.rest.resource.project;
+
+public enum ProjectReadAccessType
+{
+    PUBLIC, PRIVATE, CUSTOM;
+
+    public static ProjectReadAccessType from( final String value )
+    {
+        if ( value.equalsIgnoreCase( PUBLIC.toString() ) )
```

Comment:
@vbradnitski the `value` can be `NULL`, will be better to use `PRIVATE.name().equals(value)` or `Objects.equals(PRIVATE.name(), value)`. Also you can use `ProjectReadAccessType.valueOf(...)`

Reply:
yeah. equalsIgnoreCase is a smelling idea as well.